katsdpimager/render.py

```diff
@@ -0,0 +1,142 @@
+"""Render images using matplotlib."""
+
+import numpy as np
+from astropy import units
+from astropy.wcs import WCS
+import astropy.io.fits as fits
+import matplotlib
+matplotlib.use('Agg')     # noqa: E402
+import matplotlib.axes
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+import katsdpsigproc.zscale as zscale
+
+
+def _prepare_axes(wcs, width, height, image_width, image_height):
+    DPI = 96
+    fig = plt.Figure(figsize=(width / DPI, height / DPI), dpi=DPI)
+    ax = fig.add_subplot(projection=wcs, slices=('x', 'y', 0, 0))
+    ax.set_xlabel('Right Ascension')
+    ax.set_ylabel('Declination')
+    ax.set_xlim(-0.5, image_width - 0.5)
+    ax.set_ylim(-0.5, image_height - 0.5)
+    return fig, ax
+
+
+def _plot(hdus, caption, ax, extent, vmin, vmax):
+    data = hdus[0].data[0, 0, :, :]
+    if not ax.images:
+        im = ax.imshow(data, origin='lower', cmap='afmhot', aspect='equal',
+                       vmin=vmin, vmax=vmax, extent=extent)
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes('right', pad='3%', size='5%', axes_class=matplotlib.axes.Axes)
+        cbar = ax.get_figure().colorbar(im, cax=cax, orientation='vertical')
+        cbar.set_label('Jy/beam')
```

Comment:
The 'Jy/beam' label doesn't appear in the example figure you sent me. Perhaps you have to make the figure slightly wider for it to be visible?

Also, it might be better to scale the image units to mJy/beam because most (if not all) of the time the zscale algorithm will find limits at the mJy level or below for any reasonable (i.e. >2hr) observation length. A label of '1 mJy/beam' is easier to read than a label of '0.001 Jy/beam', and will take up less space on the plot. Better still you could try and auto-detect the most sensible units to use based on the values of `vmax` and `vmin`.